src/objective_types/oncedifferentiable.jl

```diff
@@ -78,7 +78,7 @@ function OnceDifferentiable(f, x_seed::AbstractArray, F::AbstractArray, DF::Abst
         fdfF = make_fdf(f, x_seed, F)
         return OnceDifferentiable(fF, dfF, fdfF, x_seed, F, DF)
     else
-        F2 = similar(F)
+        F2 = copy(F)
```

Comment:
Is `F2` mutated in the `jacobian!` call below? Would it still work if `F2` can't be mutated anymore? `similar` is guaranteed to create a mutable array whereas `copy` may return a static array.

Reply:
Fixed in facf227. I attempted to add a test but couldn't figure out how to make it fail outside of LsqFit so I added a comment instead.